src/css/components/_form.scss

```diff
@@ -24,5 +26,15 @@
     font-size: clamp(1rem, 1rem + 1vw, 2rem);
     border: 0.125rem solid var(--text-color);
     border-radius: 0.25rem;
+
+    &--mini {
+      max-width: 8ch;
```

Comment:
TIL: The `ch` CSS unit.

Reply:
One of the best units for limiting line lengths!